FWCore/Catalog/test/FileLocator_t.cpp

```diff
@@ -2,50 +2,52 @@
 #include "catch2/catch_all.hpp"
 
 #include <array>
+#include <memory>
 #include <string>
+#include <vector>
 
 #include "FWCore/Catalog/interface/FileLocator.h"
 #include "FWCore/Catalog/interface/SiteLocalConfig.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
 #include "FWCore/ServiceRegistry/interface/ServiceRegistry.h"
 #include "FWCore/ServiceRegistry/interface/ServiceToken.h"
 
 #include "TestSiteLocalConfig.h"
```

Comment:
I _beliebe_ the `TestSiteLocalConfig` could be removed with these changes. Could you try that?